src/Microsoft.Health.SqlServer/Features/Schema/Manager/SchemaClient.cs

```diff
@@ -49,7 +49,7 @@ public async Task<string> GetScriptAsync(int version, CancellationToken cancella
         }
         else
         {
-            throw new SchemaManagerException(string.Format(CultureInfo.CurrentCulture, Resources.ScriptNotFound, response.StatusCode));
+            throw new SchemaManagerException(Resources.ScriptNotFound);
```

Comment:
Some of these resources were not actually format strings and so these `string.Format` calls did not do anything